### PR TITLE
Docs aws modis

### PR DESCRIPTION
**Pull request by thaisbendixen42**
Pull Request
============

> Tracked in JIRA by [IN-XXX](https://geoinformationstore.atlassian.net/browse/IN-XXX)
> Paired with @a, @b

### Scope of the PR

Describe the scope of this pull request. Is this a new feature or a bug fix?

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly
- [ ] Changed links -> prod marketplace link needs to be added

### CODE REVIEW

File: source/up42-blocks/data/aws-modis.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+.. _aws-modis-fullscene-block:
+
+AWS Modis MCD43A4
+=================
+Please see the `block details page <https://marketplace.up42.com/block/98c1acfa-c141-4095-b2e6-acf52d110178>`_ for context.
+
+Block type: ``DATA``
+
+This block provides full scenes of Moderate Resolution Imaging Spectroradiometer (MODIS) MCD43A4 Version 6 Nadir Bidirectional
```
Comment (chrieke) [inline source/up42-blocks/data/aws-modis.rst:13]: handful :) Maybe first say "full scenes of MODIS MCD43A4 product. This is the Moderate Resolution Imaging Spectroradiometer (MODIS)  MCD43A4 Version 6 Nadir Bidirectional ...

File: source/up42-blocks/data/aws-modis.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+Reflectance Distribution Function (BRDF)-Adjusted Reflectance (NBAR). The data is collected from `AWS MODIS bucket <https://registry.opendata.aws/modis-astraea/>`_
+in TIF format is produced daily using 16 days of Terra and Aqua MODIS data at 500 meter (m) resolution. The view angle effects are removed from the directional reflectances,
```
Comment (chrieke) [inline source/up42-blocks/data/aws-modis.rst:15]:
```suggestion
in TIF format. The image products are produced daily using 16 days of Terra and Aqua MODIS data at 500 meter (m) resolution. The view angle effects are removed from the directional reflectances,
```

File: source/up42-blocks/data/aws-modis.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+resulting in a stable and consistent NBAR product.
+
+Important application areas for MODIS MCD43A4 imagery are: land cover monitoring (agriculture, forestry), inland water monitoring and flood mapping.
+
+Supported query parameters
+--------------------------
+
+For more information on supported filters, see :ref:`query filter section  <filters>`.
+
+* ``ids``: MODIS product IDs. e.g. ``["MCD43A4.A2018212.h23v05.006.2018226175455",...,"MCD43A4.A2018213.h23v05.006.2018226181629"]``
```
Comment (chrieke) [inline source/up42-blocks/data/aws-modis.rst:25]: Maybe add link to a ressource with the schema from nasa etc?

File: source/up42-blocks/data/aws-modis.rst [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+            {
+                "ids": [
+                    "MCD43A4.A2020182.h10v06.006.2020191030645"
+                ]
+            }
+    }
+
+Output format
+-------------
+The output data is stored in a folder with same name as the Product ID. This folder contains total of 12 GeoTIFF files and 3 metadata files.
```
Comment (chrieke) [inline source/up42-blocks/data/aws-modis.rst:78]:
```suggestion
The output data is stored in a folder with the same name as the Product ID. This folder contains total of 12 GeoTIFF files and 3 metadata files.
```

Comment (thaisbendixen42) [reply source/up42-blocks/data/aws-modis.rst:78]: Thanks for the suggestions! On it! :)

File: source/up42-blocks/data/aws-modis.rst [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+AWS Modis MCD43A4
+=================
+Please see the `block details page <https://marketplace.up42.com/block/98c1acfa-c141-4095-b2e6-acf52d110178>`_ for context.
+
+Block type: ``DATA``
+
+The block provides full scenes of the MODIS MCD43A4 product. This is the Moderate Resolution Imaging Spectroradiometer (MODIS) MCD43A4 Version 6 Nadir Bidirectional
+Reflectance Distribution Function (BRDF)-Adjusted Reflectance (NBAR) product. The data is collected from `AWS MODIS bucket <https://registry.opendata.aws/modis-astraea/>`_
+in TIF format. The image products are produced daily using 16 days of Terra and Aqua MODIS data at 500 meter (m) resolution. The view angle effects are removed from the directional reflectances,
```
Comment (perusio) [inline source/up42-blocks/data/aws-modis.rst:15]: Is not TIF is TIFF and I suppose it is a GeoTIFF not a regular TIFF.

File: source/up42-blocks/data/aws-modis.rst [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+* Individual band files are suffixed with ``<productId>_B{1..11}.TIF``
+* Quality assessment band ``<productId>_BQA.TIF``
+* Overview file for each .TIF ``<productId>_.TIF.ovr``
+* Metadata files ``data.json``, ``<productId>.hdf.xml`` and ``<productId>_meta.json``
+* RGB composite file ``<productId>.1.jpg``
+* Links to the GeoTIFFs ``index.html``
+
+
+.. note::
+  Although the output files are GeoTIFF, we refer to the whole bundle as ``MTL`` format!
```
Comment (perusio) [inline source/up42-blocks/data/aws-modis.rst:89]: No exclamation mark here please. Is just a normal statement.

File: source/up42-blocks/data/aws-modis.rst [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+AWS Modis MCD43A4
+=================
+Please see the `block details page <https://marketplace.up42.com/block/98c1acfa-c141-4095-b2e6-acf52d110178>`_ for context.
+
+Block type: ``DATA``
+
+The block provides full scenes of the MODIS MCD43A4 product. This is the Moderate Resolution Imaging Spectroradiometer (MODIS) MCD43A4 Version 6 Nadir Bidirectional
+Reflectance Distribution Function (BRDF)-Adjusted Reflectance (NBAR) product. The data is collected from `AWS MODIS bucket <https://registry.opendata.aws/modis-astraea/>`_
+in TIF format. The image products are produced daily using 16 days of Terra and Aqua MODIS data at 500 meter (m) resolution. The view angle effects are removed from the directional reflectances,
+resulting in a stable and consistent NBAR product.
```
Comment (perusio) [inline source/up42-blocks/data/aws-modis.rst:16]: What is NBAR? Context link?

File: source/up42-blocks/data/aws-modis.rst (hunk 3 above)
Comment (perusio) [reply source/up42-blocks/data/aws-modis.rst:25]: Is there a schema description anywhere for this?

**Pull request merged**